utils/merkletrie/fsnoder/file_test.go

```diff
@@ -0,0 +1,67 @@
+package fsnoder
+
+import (
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"gopkg.in/src-d/go-git.v4/utils/merkletrie/noder"
```

Comment:
This should be in reverse order, first a block with go-git imports, then another with external dependencies.